environment.yml

```diff
@@ -8,7 +8,6 @@ dependencies:
   - pip
   - pytorch
   - torchvision
-  - cudatoolkit=10.2
```

Comment:
Can we make a separate `environment_cpu.yml` file?

Reply:
done